README.md

```diff
@@ -1,24 +1,249 @@
-# README
+<a name="readme-top"></a>
```

Comment:
### I'm just using the readme file to write comments because not all files are included in this PR you should modify the code that is responsible for what is shown below 

- One of the requirements is to follow the given design by implementing the same colors, font-family, spaces, and layout you can see the screenshot of the requirements below 
<img width="546" alt="Screenshot 2023-02-15 at 15 24 42" src="https://user-images.githubusercontent.com/61790587/219056526-1c6f88ad-759e-45a9-805d-f8aa0f2e4ded.png">

- [ ] it would be better to stick to the design make the transaction  page the same as in the design by adding the same font size, and spaces, and layout and making sure the add button should be the same as the one in the design 
- you should use the same colors also 

- Transaction page design 
<img width="233" alt="Screenshot 2023-02-15 at 15 44 21" src="https://user-images.githubusercontent.com/61790587/219060085-5b2d98eb-72f6-4c8e-bdae-997f0fcb4984.png">
<img width="233" alt="Screenshot 2023-02-15 at 15 40 46" src="https://user-images.githubusercontent.com/61790587/219060208-e37dc808-ee6c-4841-9c68-56487b81281e.png">



- Yours 
![Screenshot 2023-02-15 at 15 44 30](https://user-images.githubusercontent.com/61790587/219060153-9d35346c-c229-4726-8286-18315f4ba3bc.png)
![Screenshot 2023-02-15 at 15 44 09](https://user-images.githubusercontent.com/61790587/219060252-2ec1daf8-69cf-4480-ba3c-3dca5b0d20b7.png)
- don't forget to use the same color as well


- Font family from the Design 
<img width="412" alt="Screenshot 2023-02-15 at 15 25 53" src="https://user-images.githubusercontent.com/61790587/219057802-99df8d3a-3e32-4603-badf-f2e71e8bd4c0.png">